src/app/Library/CrudPanel/Uploads/RegisterUploadEvents.php

```diff
@@ -0,0 +1,183 @@
+<?php
+
+namespace Backpack\CRUD\app\Library\CrudPanel\Uploads;
+
+use Backpack\CRUD\app\Library\CrudPanel\CrudColumn;
+use Backpack\CRUD\app\Library\CrudPanel\CrudField;
+use Backpack\CRUD\app\Library\CrudPanel\CrudPanelFacade as CRUD;
+use Backpack\CRUD\app\Library\CrudPanel\Uploads\Interfaces\RepeatableUploaderInterface;
+use Backpack\CRUD\app\Library\CrudPanel\Uploads\Interfaces\UploaderInterface;
+use Exception;
+
+final class RegisterUploadEvents
```

Comment:
Haha I was just going to suggest we make this class `final`. Well done! 👏👏👏

We should do this more going forward.